canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-builder.scss

```diff
@@ -47,7 +47,13 @@ $operator-margin: 50%;
 	margin-bottom: 0;
 
 	.elyra-CodeMirror {
+		/* Allow codemirror container to resize on drag */
+		resize: vertical;
+		overflow: auto;
 		.cm-editor {
+			min-height: inherit; /* Inherit min-height to restrict resize below certain width  */
+			resize: vertical; /* Allow codemirror texteditor to resize on drag */
```

Comment:
I removed the resize and overflow here and it works correctly.
https://github.com/user-attachments/assets/5864a04a-7ede-4920-b704-1bb9efe474c1

Reply:
Yes, its working without that I have updated the PR. Thanks.
For some reason it was behaving like above recording now when I restart the browser its working fine.